SpywareFilter/sections/tracking_servers.txt

```diff
@@ -4888,6 +4888,8 @@
 ! note that we don't use || prefixes because mail tracking pixels are often proxied by mail providers
 ! it's problematic that due to how Gmail proxy work, these domains aren't even exposed on the network level
 ! fortunately, extensions can see them (Gmail puts the real URL in the location.hash part of the src)
+! Shutterstock
+shutterstockmail.com/pub/as?_ri_=$image
```

Comment:
Could you provide an URL or screenshot from the network tools, where you have this request?